test/Serialization/group_info_diags.swift

```diff
@@ -4,7 +4,16 @@
 // RUN: not %target-swift-frontend -emit-module %s -module-name HasArray -o %t -module-cache-path %t/mcp -group-info-path %S/Inputs/corrupted_group_info.json -emit-module-doc &> %t/result.txt
 // RUN: %FileCheck %s -check-prefix=CORRUPTED < %t/result.txt
 
-public protocol P {}
+// RUN: %target-swift-frontend -emit-module %s -module-name HasArray -o %t -module-cache-path %t/mcp -group-info-path %S/Inputs/group_info.json -emit-module-doc &> %t/result.txt
+// RUN: strings %t/HasArray.swiftdoc > %t/doc_strings.txt
```

Comment:
@shahmishal @compnerd should use of `strings` binary in the test be guarded with `// REQUIRES: shell` ?

Reply:
No, you should use `llvm-strings` instead.

Reply:
Ah nice, I wasn't aware of this, @nkcsgexi could you make such a change ?

Reply:
sure, I have cherry-picked @compnerd 's https://github.com/apple/swift/pull/24092 here